client/a8c-for-agencies/sections/marketplace/wpcom-overview/wpcom-slider/style.scss

```diff
@@ -27,6 +27,10 @@
 		cursor: default;
 	}
 
+	&:read-only::-moz-range-thumb {
+		cursor: default;
+	}
```

Comment:
[Nitpick] We could combine this instead of creating a new one.

```
&:read-only::-webkit-slider-thumb, &:read-only::-moz-range-thumb  {
   cursor: default;
}
```

Reply:
For some reason, this doesn't work. I have to separate this into two rule scopes. Similar to the existing `&::-webkit-slider-thumb` and `&::-moz-range-thumb`.